test/test_helper.rb

```diff
@@ -1,31 +1,16 @@
-require 'codeclimate-test-reporter'
-CodeClimate::TestReporter.start
-require 'simplecov'
-SimpleCov.start
-
 # Configure Rails Environment
 ENV['RAILS_ENV'] = 'test'
 
+require 'codeclimate-test-reporter'
+require 'simplecov'
+require 'byebug'
+
 require File.expand_path('../../test/dummy/config/environment.rb', __FILE__)
-ActiveRecord::Migrator.migrations_paths = [File.expand_path('../../test/dummy/db/migrate', __FILE__)]
-ActiveRecord::Migrator.migrations_paths << File.expand_path('../../db/migrate', __FILE__)
 require 'rails/test_help'
 require 'minitest/rails'
 require 'minitest/reporters'
-Minitest::Reporters.use!
-
-# Filter out Minitest backtrace while allowing backtrace from other libraries
-# to be shown.
-Minitest.backtrace_filter = Minitest::BacktraceFilter.new
-
-# Load support files
-Dir["#{File.dirname(__FILE__)}/support/**/*.rb"].each { |f| require f }
-
-# Load fixtures from the engine
-if ActiveSupport::TestCase.respond_to?(:fixture_path=)
-  ActiveSupport::TestCase.fixture_path = File.expand_path('../fixtures', __FILE__)
-  ActionDispatch::IntegrationTest.fixture_path = ActiveSupport::TestCase.fixture_path
-  ActiveSupport::TestCase.fixtures :all
-end
-
 require 'webmock/minitest'
+
+#Minitest::Reporters.use!
```

Comment:
I just found it difficult to parse while writing tests so changed it back. If it's required to have then I can add them back. There is a number of other things I should probably clean up

* byebug_history etc.